apps/dav/lib/CardDAV/ImageExportPlugin.php

```diff
@@ -90,7 +90,7 @@ public function httpGet(RequestInterface $request, ResponseInterface $response)
 
 			$response->setBody($file->getContent());
 		} catch (NotFoundException $e) {
-			$response->setStatus(404);
+			$response->setStatus(204); // 204 No Content
```

Comment:
```suggestion
			$response->setStatus(\OCP\AppFramework\Http::STATUS_NO_CONTENT);
```

Please consider to import the class.

Reply:
Why would it be a problem to use`\OCP\AppFramework\Http`?

- The status code is the same
- The http class is from our app framework and not specific to SabreDAV or Symfony